source/copilot-docs.txt

```diff
@@ -0,0 +1,69 @@
+.. _vsce-copilot-docs:
+
+==============
+/docs Command
+==============
+
+.. default-domain:: mongodb
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+The ``/docs`` command provides MongoDB-specific information,
+supplemented by links to MongoDB Documentation. The |copilot| is
+knowledgeable of the latest MongoDB Documentation and uses the
+MongoDB Docs Chatbot API to link to information in addition to
```

Comment:
```suggestion
MongoDB Docs Chatbot to link to information in addition to
```
I don't believe the API is intended for use outside of our use cases; we don't want folks using it at the moment. Maybe we can leave the API part out so folks don't get ideas?

Reply:
Removed the mention of the API!